understanding/20/focus-order.html

```diff
@@ -64,30 +63,54 @@ <h2>Intent of Focus Order</h2>
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
       
+      <p>Focus order needs to allow the user to navigate focusable elements in a logical order,
+         and that order needs to preserve any meaning or operation that the page is conveying. 
+         Focus order does not necessarily need to follow the visual layout of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, the assumption in western left-to-right systems may be that focus moves through the elements in the
+         left-hand column first, from the top to the bottom of the column, and then proceed to the right-hand column.
+         However, if the two columns are independent of each other, and meaning/operation are not affected, it is not
+         a failure if elements in the right-hand column receive focus first, followed by the elements in the left-hand column.
+         As a best practice, though, make sure the focus order reinforces the reading order implied by the visual layout.
+      </p>
+      
+      <div class="note">
+         <p>It is not a failure of this Success Criterion if static/non-interactive content
+         receives focus. Making an element that does not act as an actionable user interface component
+         focusable, or programmatically moving focus to such an element, does not necessarily constitute
+         a failure of this Success Criterion. This situation is a failure if these additional focus stops
+         impede the operation of the content, or create confusing and/or illogical focus orders - for example,
```

Comment:
I whole heartedly agree with this proposed language, however I've found that testers have trouble with determining when additional focus stops impede the operation of the content. For example, is just one unnecessary tab stop enough to say that it impedes the operation? 3? 6? Does it depend on context? It may be worthwhile to establish some sort of bar/test for which to determine this consistently. I don't have an immediate suggestion, but I'll think on it.

Reply:
Perhaps I should rea the convo in the pull request before posting a comment directly in the file. I see that there is already discussion on this. 😅